docs/application/dotnet/guides/internationalization/localization.md

```diff
@@ -0,0 +1,252 @@
+Tizen uses res.xml file to specify the information about the directory, which contains the localized resources(For example, Image, Sound, and so on).
+The res.xml file is automatically generated when you build your application.
+
+**To add the localized resource directories in your application in Visual Studio:**
+
+- In Solution Explorer, select Tizen project. Go to **Tools &gt; Tizen &gt; Resource Manager**
+
+![local_res_mgr_1](media/local_res_mgr_1.png)
+
+- Resource Manager window appears. In Configuration tab, select from the language drop-down list.
+
+![local_res_mgr_2](media/local_res_mgr_2.png)
+
+- Select the language, Click **Add**.
+
+![local_res_mgr_3](media/local_res_mgr_3.png)
+
+![local_res_mgr_4](media/local_res_mgr_4.png)
+
+- Resource directories are automatically created in Tizen project as:
+  res.xml file will be generated in res directory of Tizen project after building an application.
+![res.xml](media/local_res_xml.png)
+![res.xml code](media/local_res_xml_code.png)
+
+> **Note**
+>
+>Your application can sometimes run in a locale, for which you have not provided images. In that case, Tizen loads the default image from the resource content directory (yourApp.Tizen/res/content/). If there is no default image within resource content directory and the device sets the locale, for which you have not provided images, an error occurs.
+
+![default_image](media/local_res_default_image.png)
+
+When you detect locale changes, you must update the resource culture. Then you must update the texts and images, which you want to localize.
+As for localized images, you can make custom image renderer to load the proper locale-specific images.
+Tizen provides the path of locale-specific images via `ResourceManager.TryGetPath` and `ResourceManager.GetPath` methods. With this, you can change
+
+**class LocalizedImageRenderer :ImageRenderer**
+```cs
+        using TizenResourceManager = Tizen.Applications.ResourceManager;
+  
+        public LocalizedImageRenderer() : base()
+        {
+            SystemSettings.LocaleLanguageChanged += SystemSettings_LocaleLanguageChanged;
+        }
+ 
+        // Invoked every time the language setting has been changed
+        private void SystemSettings_LocaleLanguageChanged(object sender, LocaleLanguageChangedEventArgs e)
+        {
+            // Get the path of a proper image based on locale and update the source of an image
+            Element.Source = TizenResourceManager.TryGetPath(TizenResourceManager.Category.Image, fileName);
+        }
+```
+
+## Application Name Localization
+
+You can add localized application name and icon using **tizen-manifest.xml** in the Tizen project.
+
+- Open the **tizen-manifest.xml**, select the **Localization** tab and click **Add** to add **Name**.
+
+- Add application names for languages you want to support.
+
+![app_name](media/local_application_name.png)
+
+## Related Information
+* Dependencies
```

Comment:
Please use "-" to make a list instead of "*".

Reply:
done